modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/init.es.js

```diff
@@ -124,6 +124,22 @@ const initSPA = function () {
 		});
 	};
 
+	delegate(document, 'click', 'a', (event) => {
```

Comment:
This feels reasonable-ish... it might make sense to move this to the `onDocClickDelegate_` in `App.es.js`

Reply:
Yes, It feels more okay on onDocClickDelegate_ and We could use `Liferay.SPA.app.linkSelector` instead of just `a`